specification/Metadata/0.0.0/README.md

```diff
@@ -74,50 +138,75 @@ Using terminology from the geospatial industry, **features** are geometric entit
 
 This specification adds a mechanism for storing metadata in existing 3D model formats such as Khronos Group's glTF or Cesium's 3D Tiles. Metadata can be applied at various granularities: per-vertex, per-texel, as well as larger structures from meshes to tilesets. This metadata can be used for analytics purposes, as well as styling in a 3D renderer.
 
-Guiding principles for this specification include:
+Guiding principles include:
 
 - Design a data format that allows for runtime efficiency, even for large datasets.
-- Keep class definitions separate from instantiation. This allows for greater flexibility of data formats, and allows for reuse of metadata definitions.
+- Keep class definitions separate from instantiation. This allows for greater flexibility of data formats, and allows for reuse of metadata definitions.s
+
+For a complete list of vocabulary used, see the [Glossary](#glossary)
 
 ## Concepts
 
-### Classes
+### Data Types
 
-A **class** describes a collection of related pieces of metadata called **properties** and their respective data types. In some ways, this is similar to describing the columns of a database table. However, this analogy is not completely applicable, as this specification allows for texture storage, not just columnar formats.
+**Primitive types** are data types that cannot be divided further. These include:
```

Comment:
Are we okay with calling the smallest metadata types (numeric and strings) "primitive types"? I also considered "intrinsic", but I think "primitive" better conveys "you can't divide this further"

(Yes I did consider "atomic" as well, but that has a different meaning in concurrent programming)